portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -21,17 +21,18 @@
 
 @WebServlet("/list_comments")
 public class ListCommentsServlet extends HttpServlet{
-
     private static final String NAME = "name";
     private static final String COMMENT = "comment";
+    private static final String IMAGEURL = "imageUrl";
```

Comment:
IMAGE_URL

Reply:
when is camelCase better and when underscores?

Reply:
java style dictates that all variable names are camelCase *except* constants, which are all uppercase and therefore need to use underscores.